lib/trento/hosts/host.ex

```diff
@@ -738,17 +802,30 @@ defmodule Trento.Hosts.Host do
   defp compute_tuning_health("compliant"), do: Health.passing()
   defp compute_tuning_health(_), do: Health.unknown()
 
+  defp compute_software_updates_discovery_health(%RelevantPatches{total: 0}),
+    do: Health.passing()
+
+  defp compute_software_updates_discovery_health(%RelevantPatches{
+         security_advisories: security_advisories
+       })
+       when security_advisories > 0,
+       do: Health.critical()
+
+  defp compute_software_updates_discovery_health(_), do: Health.warning()
```

Comment:
I'm just curious as to when `Health.warning()` triggers?

Reply:
> warning if there are only buxfixes/software enhancements
- no patches -> passing
- some patches and at least one security advisory -> critical
- some patches, no security advisories but only buxfixes/software enhancements -> warning